docs/src/api_comparison.md

```diff
@@ -11,96 +11,98 @@ welcome.
 | Operation                             | JuliaImages                               | scikit-image + NumPy              | Matlab (ImageProcessing + ComputerVision) |
 |---------------------------------------|-------------------------------------------|-----------------------------------|-------------------------------------------|
 | **Input/output**                      |                                           |                                   |                                           |
-| Read image file                       | `load` (FileIO.jl)                        | `imread`                          | `imread`                                  |
-| Write image file                      | `save` (FileIO.jl)                        | `imsave`                          | `imwrite`                                 |
+| Read image file                       | [`load`](@ref)                            | `imread`                          | `imread`                                  |
```

Comment:
Having these link to the reference page will be a huge usability improvement! :fireworks: